smart-contract/contracts/YourNftToken.sol

```diff
@@ -68,15 +73,21 @@ contract YourNftToken is ERC721AQueryable, Ownable, ReentrancyGuard {
     _safeMint(_msgSender(), _mintAmount);
   }
   
-  function mintForAddress(uint256 _mintAmount, address _receiver) public mintCompliance(_mintAmount) onlyOwner {
+  function mintForAddress(uint256 _mintAmount, address _receiver) public onlyOwner {
```

Comment:
I'm curious why you removed `mintCompliance` modifier from this function?

Also, if this is ok shouldn't you have the other require there? i.e:

```
function mintForAddress(uint256 _mintAmount, address _receiver) public onlyOwner { 
    require(_mintAmount > 0 && _mintAmount <= maxMintAmountPerTx, 'Invalid mint amount!');
    require(totalSupply() + _mintAmount <= maxSupply, 'Max supply exceeded!');

    _safeMint(_receiver, _mintAmount);
}
```

Reply:
This is unrelated to the Opernsea filter, just a general tweak that I have found people have issues with sometimes. I did briefly mention it in the push, but this change simply removes the restriction of the owner to mint more than the current maxMintAmountPerTx. It still ensure that maxSupply will not be exceeded, but allows the contract owner to mint multiple tokens, even if maxMintAmountPerTx is set to 1, for example.